### PR TITLE
Add React standalone Interface Editor

### PR DESCRIPTION
**Pull request by davidebriani**
This PR adds a React implementation for the standalone Interface Editor and use React to render this page when no app config is present.
Consequently, the previous Elm implementation of the Interface Editor is removed since it's now unused.

### CODE REVIEW

File: src/static/index.js [hunk 1]
```diff
@@ -82,39 +82,43 @@ function clearReact() {
...
-    appengineUrl: conf.appEngineApiUrl,
-    pairingUrl: conf.pairingApiUrl,
-    flowUrl: conf.flowApiUrl,
-    enableFlowPreview: conf.enableFlowPreview,
-  });
-
-  if (sessionManager.isUserLoggedIn) {
-    astarteClient.setCredentials(sessionManager.getCredentials());
-  }
+  if (!dashboardConfig) {
```
Comment (Pavinati) [inline src/static/index.js:85]: This little change creates a huge diff where everything is identical, just indented one layer deeper.
What about an early return?
```
if (!dashboardConfig) {
  app = elmApp.init();
  return;
}
```

Comment (davidebriani) [reply src/static/index.js:85]: I just didn't want to duplicate the code needed to setup Elm ports, which follows the if-else statement.

Comment (Pavinati) [reply src/static/index.js:85]: ports are just for page changes, but there should be no page changes in the editor only mode

Comment (davidebriani) [reply src/static/index.js:85]: A quick experiment confirmed that those lines are also needed to render the react page, even if there is no subsequent route change.

**Review by Pavinati — APPROVED** [review]

**Pull request merged**